docs/docs/propeller/index.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 4
+---
+
+# Propeller
+
+Propeller is EquinoxJS's "[propulsion](https://github.com/jet/propulsion)
+lite". It's a set of abstractions for building reactive event processing
```

Comment:
the abstractions are not the sell really - its the fact that there are impls, and that they have a ecent structure that will yield you a testable/separated handler not mangled in store specific bpoilerplate.

It's a set of components for building ...
The primary purpose is to manage reactions to events emanating from an event store's feed.
The components all follow common high level patterns, with specific treatments and options to allow you to manage specific details of the actual store you're using.
The high level structure is also designed to work well in terms of being able to test handlers in isolation, or deal with similar needs, such as continually ingesting data from third party event feeds.

(last point is probably not intro material)

Reply:
You are absolutely correct. It's just so much easier to write about the abstractions :(